src/cdk/drag-drop/drag-utils.ts

```diff
@@ -51,6 +50,26 @@ export function transferArrayItem<T = any>(currentArray: T[],
   }
 }
 
+/**
+ * Copies item from one array to another, leaving copied item in its
```

Comment:
copies item -> copies an item
leaving item -> leaving it

Also missing a full stop at the end.